frontends/api/src/hooks/articles/keyFactory.ts

```diff
@@ -5,8 +5,10 @@ import { createQueryKeys } from "@lukemorales/query-key-factory"
 const articles = createQueryKeys("articles", {
   detail: (id: number) => ({
     queryKey: [id],
-    meow: "meow",
```

Comment:
what was `meow` for? 😆

Reply:
😆 I vaguely remember for some reason I was trying to see if we could other data on these factories... "meow" and "woof" are my go-to foo bars 😆.